packages/wonder-blocks-icon-button/src/components/node-icon-button.tsx

```diff
@@ -10,21 +10,39 @@ import {border, semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
 import type {BaseIconButtonProps} from "../util/icon-button.types";
 
 import {IconButtonUnstyled} from "./icon-button-unstyled";
+import {
+    mapTokensToVariables,
+    TokensAsCssVariable,
+    TokensAsJsVariable,
+} from "../util/map-tokens-to-variables";
 
 /**
- * The object containing the CSS variables that can be overridden to customize
- * the appearance of the NodeIconButton component.
+ * The prefix for the CSS variables used in the NodeIconButton component.
+ *
+ * This allows us to avoid collisions with other CSS variables in the
+ * application.
+ */
+const VAR_PREFIX = "--wb-c-node-icon-button--";
```

Comment:
Should we use VAR_PREFIX where we set the css variables? I think there's a case where the dynamically generated styles with the overriding css variables can get out of sync with the css variables defined in the component!